app/helpers/application_helper.rb

```diff
@@ -1,2 +1,13 @@
 module ApplicationHelper
+
+  def logged_in?
```

Comment:
@JuanMaGiraldo I think this can be removed due to those methods are already placed on the Application controller, am I right?